### PR TITLE
fix: escaped comment interpolation in error message

### PR DESCRIPTION
**Pull request by ActuallyHappening**

### CODE REVIEW

**Comment by ogoffart** [comment]
thanks!
could you fix the test faillure?

**Comment by ActuallyHappening** [comment]
This actually exposed a subtle 'bug' (which I just ignored),
the test case required adding two spaces and a `\n`

works, and is better diagnostic

File: lib.rs [hunk 1]
```diff
@@ -343,7 +343,7 @@ fn process_toml(cargo_toml: &str, args: &Args) -> Result<String, String> {
                     }
                 } else if current_table != "features" {
                     return Err(format!(
-                        "Comment cannot be associated with a feature: '{}'",
+                        r#"Comment cannot be associated with a feature: "{}""#,
                         current_comment
```
Comment (ogoffart) [inline lib.rs:347]: Maybe the error would look better if we trim the whitespace out of current_comment?

Comment (ActuallyHappening) [reply lib.rs]: yes, see the next commit I did
has the exact same idea

**Comment by ogoffart** [comment]
Thanks!

**Pull request merged**